javascript-client/tutorial.md

```diff
@@ -266,24 +266,46 @@ Also, our sample app works directly with Metamask using `windows.ethereum`
 
 1. First add GSN to your project:
 ```
-yarn add @opengsn/provider @opengsn/contracts @opengsn/dev
+yarn add @opengsn/provider @opengsn/contracts
+```
+
+Also you will need the following dev dependencies to run your project locally:
+```
+yarn add --dev @opengsn/dev @opengsn/cli
 ```
 
 2. When you test your project locally, you usually start a local node using `hardhat node`.
-   To start a node with GSN deployed (and a relayer running), you should run:
+In order to start a Hardhat node run:
+```
+npx hardhat node
+```
+
+_Note that using `hardhat-deploy` plugin with the GSN is currently not supported. The GSN requires the Hardhat to run in a
```

Comment:
Its not that GSN is incompatible - we only need to prevent using a forwarder during deployment. before GSN is started
there are 2 options:
1. manual startup:
- hardhat node --no-deploy
- gsn start
- hardhet deploy --network dev

2. late-assignment of forwrader:
- hardhat node (which does "deploy")
- gsn start
- custom script to call setForwarder

3. deterministic forwarder
- hardhat-deploy
- gsn start

(the only required feature is add "det. deployment" to forwrader in "dev" mode, which we currently don't have.

Reply:
Ok, I will document the first one then. Won't say "incompatible" but will try to explain the problem.